### ISSUE TITLE
Not working for some valid CSV files

### ISSUE THREAD

**Issue by nurbles** [BdR76/CSVLint#36]
I'm attaching a couple of valid CSV files that CSV Lint is telling me have only one column:

[dummy4.csv](https://github.com/BdR76/CSVLint/files/9478496/dummy4.csv)
[dummy3.csv](https://github.com/BdR76/CSVLint/files/9478497/dummy3.csv)

MS Excel opens them correctly, without complaint, as do all of the other CSV-aware programs I could find.  I though the issue was that the header row didn't have the field names quoted, but neither quoting the header field names nor removing the header row entirely resolved the issue.  I also tried removing the empty field at the end of the records (and its header field) but that didn't help, either.

[EDIT:  I'm using v0.4.5.2 in NPP v8.4.4 (64-bit)]
[EDIT2:  I discovered that there is a newer version, v0.4.5.5β, so I installed and tried it, but this issue persists.]

**Comment by BdR76** [comment]
Thanks for posting the feedback and the example files. The problem is that the plug-in doesn't skip empty lines when automatically detecting the separator character. This is a bug and I'll fix it in the next update.

For now, you can remove the last empty lines and then close and open the file again. Or, in the latest plugin version, you can open the `CSV Lint window` then uncheck the "automatic" checkbox and press the "Detect Columns" button to manually set the Column separator to comma `,`.

**Comment by nurbles** [comment]
Thanks for the work-around tips!  Both work as described.

FYI:  When I opened the CSV Lint window, the buttons at the top,left looked like this:  [overlapping buttons](https://www.svalli.com/files/Overlap.png)

I don't know how much control you have over the width of the two panels in the CSV Lint area, but I must say, this view was a little confusing!  (smile)

Thanks again for the tips!

**Comment by BdR76** [comment]
That's good to hear. Also, thanks for mentioning the squashed buttons in the CSV Lint window, I think it has to do with the windows being resized with a lower screen resolution maybe.  I work with a regular 1920x1080 resolution and hadn't noticed this. What is the screen resolution you're working with? To see it you can right click Windows desktop, choose `Display settings` and see the current "Display resolution" setting.

Also, you can slide the window divider to make it wider, that should probably fix it, see screenshot below

![csvlint_click_and_draw](https://user-images.githubusercontent.com/7793956/188729959-94485f0a-eb65-48d7-9c9e-32f735bded8e.png)

**Comment by nurbles** [comment]
My desktop is (1) 1080x1920, (2) 3840x2160, (3) 1280x1024 with (2) being the primary screen.  This allows me to test my software on quite a few different screen resolutions & layouts directly on my development system.

For whatever reason, when I first opened the CSV Lint window, the left panel started rather narrow, with the automatic checkbox overlapping the split column button.  I dragged the sizer and discovered that the floppy disk button had been completely hidden (initially) so I resized to show a kinds of worst case example for you to see.  I intended to say it wasn't a big problem because it _could_ be resized, but since it is possible for it to open with a potentially useful button _completely hidden_ a user might not try to resize in order to see a button they never knew should be there.
